### DISCUSSION TITLE
React Native for Web

### DISCUSSION THREAD

**Discussion by axeldelafosse** [radix-ui/primitives#612] [Ideas]
Hey,

Love what you are doing here. Did you already discuss adding a package using [React Native for Web](https://github.com/necolas/react-native-web)? That would be amazing.

Happy to help!

**Comment by colmtuite** [comment]
Thanks @axeldelafosse! We've discussed releasing libs for other frameworks, but no immediate plans to focus on it just yet.

**Reply by nandorojo** [reply]
I'd like to chime in here. Adding support for React Native (and thus, React Native Web) would be an amazing addition. For context, I maintain [Dripsy](https://github.com/nandorojo/dripsy), a React Native design system based on theme-ui. We use it at [BeatGig](https://beatgig.com).

Stitches and Radix look like they are solving all the UI problems that are missing in an RN styling library. There would be massive community support and involvement for this. Chakra UI, for instance, is also planning on supporting React Native Web.

The nice thing about React Native is that the components also work nicely on web, so you're only doing the work once.
